### ISSUE TITLE
Cannot receive sms code?

### ISSUE THREAD

**Issue by StupidBug** [Hattorius/discord-account-generator#18]
It's going well before sending an SMS request, but I can't receive an SMS code anyhow, however, when I register accounts using chrome, I can receive SMS codes everytime.

**Comment by henzycuong1** [comment]
I have the same problem. can anyone give a hint ?

**Comment by StupidBug** [comment]
> I have the same problem. can anyone give a hint ?

I think it's some problem about Hcaptcha, when I block chrome SMS requests and use its captcha key, I can receive SMS code successfully, but when using captcha key from the  API,  I can't receive it

**Comment by henzycuong1** [comment]
> I think it's some problem about Hcaptcha, when I block chrome SMS requests and use its captcha key, I can receive SMS code successfully, but when using captcha key from the API, I can't receive it

yeah, I have tried on selenium and using captcha key from the API, it couldn't receive sms code. But I use selenium and manually solve captcha. It worked well.
Do you fix that problem. Can you give me a hint? Pls

**Comment by StupidBug** [comment]
> > I think it's some problem about Hcaptcha, when I block chrome SMS requests and use its captcha key, I can receive SMS code successfully, but when using captcha key from the API, I can't receive it
> 
> yeah, I have tried on selenium and using captcha key from the API, it couldn't receive sms code. But I use selenium and manually solve captcha. It worked well. Do you fix that problem. Can you give me a hint? Pls

**Comment by StupidBug** [comment]
> > > I think it's some problem about Hcaptcha, when I block chrome SMS requests and use its captcha key, I can receive SMS code successfully, but when using captcha key from the API, I can't receive it
> > 
> > 
> > yeah, I have tried on selenium and using captcha key from the API, it couldn't receive sms code. But I use selenium and manually solve captcha. It worked well. Do you fix that problem. Can you give me a hint? Pls

I don't know how to fix it yet, I tried to fix it for servel days. If you fix it please let me know!
